osutil/find_in_path_test.go

```diff
@@ -0,0 +1,77 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2017 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package osutil_test
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/snapcore/snapd/osutil"
+
+	. "gopkg.in/check.v1"
+)
+
+type findInPathSuite struct {
+	basePath string
+	envPath  string
+}
+
+var _ = Suite(&findInPathSuite{})
+
+func (s *findInPathSuite) relocatePath(path string) string {
+	return filepath.Join(s.basePath, path)
+}
+
+func (s *findInPathSuite) SetUpSuite(c *C) {
+	p, err := ioutil.TempDir("", "find-in-path")
```

Comment:
Isn't that what ioutil.TempDir does internally for me with the advantage of having a unique dir for every test run?